lib/Models/CsvCatalogItem.js

```diff
@@ -192,6 +193,20 @@ defineProperties(CsvCatalogItem.prototype, {
         get : function() {
             return CsvCatalogItem.defaultPropertiesForSharing;
         }
+    },
+
+    /**
+     * Gets or sets the tableStyle object
+     * TODO: add definition for subfields
+     * @type {Object}
+     */
+    tableStyle :  {
```

Comment:
This isn't used because the `tableStyle` defined in the constructor (with the `knockout.track` line) overrides it.